### ISSUE TITLE
Select - empty search message accessibility

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#6821]
### Expected Behavior
The empty search message should be read out by the screenreader as regular text.

### Actual Behavior
The empty search message is read out as a dimmed menu item implying that it is something that is selectable but currently disabled.

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io/?path=/story/input-select-search--search

### Steps to Reproduce
Using a screenreader do the following steps:
1. Open the select in the above story
2. type 'z' in the search
3. have the screenreader 'read all' (ctrl+option+A in voice over)

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):
